### ISSUE TITLE
Test public techtv.mit.edu with OVS back-end

### ISSUE THREAD

**Issue by pdpinch** [mitodl/odl-video-service#490]
Before we switch over the TechTV DNS, we need to manually test private URLs.

1. On the test machine, change the /etc/hosts file to map techtv to the OVS IP
2. Visit an original TechTV URL
3. Confirm:
  a. no login required
  b. comparable OVS page loads
  c. video plays
4. Test YouTube dependency by changing /etc/hosts to block YouTube, confirm:
  a. no login required
  b. comparable OVS page loads
  c. video plays
5. test video embed has similar behavior
  a. With host file settings above, visit Blossoms site
6. Review all error messages
  a. TechTV video page
  b. TechTV embed page
  c. TechTV collection page
  d. other TechTV pages: search, admin, etc.

**Comment by pdpinch** [comment]
We need at least the Blossoms public collection to be migrated in order to test this.

**Comment by emmello** [comment]
Visit an original TechTV URL
Confirm:   **Confirmed** that video techtv.mit.edu/videos/595-chandelier loads on OVS with Youtube video playing.  Navigating to the collection revrev will initiate Touchstone login.
**Note**   Also needs to be able to play from techtv.mit.edu/videos/595  - currently gives a 404 error.

a. no login required   **Confirmed login isn't required for /595-chandelier links**
b. comparable OVS page loads  **Confirmed**
c. video plays  **Video plays from Youtube**


test video embed has similar behavior
**Note**  techtv.mit.edu/embeds/595 plays the youtube video with no problem (does not have the same problem as the videos/595 404 error.)


a. With host file settings above, visit Blossoms site   - need to get blossoms videos ported over to OVS to test.

**Comment by emmello** [comment]
Added in uBlocker and blocked .youtube.com and public video plays fine from OVS:   techtv.mit.edu/videos/595-chandelier

**Comment by pdpinch** [comment]
@emmello I added another checkbox for this testing. I'd like us to review the error messages for TechTV URLs to see if they need any adjustment.

**Comment by emmello** [comment]
Test machine is now able to play either Youtube or OVS embedded videos, though both are showing a smaller window than the original TechTV Flowplayer.  Tested with Rob who noticed an opaque color in the HTML code (which I thought was on the Blossoms site) that he will eliminate on the embed code.   Playback of the OVS embed is showing black bars on either side of the videos, while the Youtube plays edge to edge.  Currently waiting changes by Rob, but functionality of playback works.

I did have a brief problem of what looked like a broken link where the embeds wouldn't play at all on the test machine.  Matt tried it in an incognito window and it worked fine.  Shortly after that, the embeds started working again so attributed the problem to a computer issue.
